File: Python/URI_judge_1040.py
nota = input().split()
n1,n2,n3,n4 = map(float,nota)

m = (n1*2.0 + n2*3.0 + n3*4.0 + n4*1.0) / 10.0
print(f"Media: {m:.1f}")

if(m >= 5.0 and m <=6.9):
    print("Aluno em exame.")
    ex = float(input())
    print(f"Nota do exame: {ex}")
    ne = (m + ex) / 2
    print("Aluno aprovado.")
    print(f"Media final: {ne:.1f}")

elif(m>=7.0):
    print("Aluno aprovado.")

else:
    print("Aluno reprovado.")

File: C/URI_judge_1040.c
#include <stdio.h>

int main(){
    double n1,n2,n3,n4,m,ex,ne;
    scanf("%lf%lf%lf%lf",&n1,&n2,&n3,&n4);
    m = (n1*2.0 + n2*3.0 + n3*4.0 + n4*1.0) / 10.0;
    printf("Media: %.1lf\n",m);

    if(m >= 5.0 && m <=6.9){
        printf("Aluno em exame.\n");
        scanf("%lf",&ex);
        printf("Nota do exame: %.1lf\n",ex);
        ne = (m + ex) / 2;
        printf("Aluno aprovado.\n");
        printf("Media final: %.1lf\n",ne);
    }

    else if (m >= 7.0){
        printf("Aluno aprovado.\n");
    }

    else{
        printf("Aluno reprovado.\n");
    }
    return 0;
}